libs/designsystem/src/lib/helpers/date-helper.ts

```diff
@@ -0,0 +1,3 @@
+export const subtractTimezoneOffset = (date: Date): Date => {
+  return new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000);
+};
```

Comment:
Is this needed anywhere else but the calendar? If not, consider inlining in CalendarComponent with rest of date manipulation methods...

Reply:
It was previously used several places, but now only in one of the examples - I'll inline it